### DISCUSSION TITLE
Word-level timestamps

### DISCUSSION THREAD

**Discussion by junwang4** [neonbjb/tortoise-tts#276] [Ideas]
Is there any way that we can get an audio as well as its word-level timestamps?

**Comment by carlosdp** [comment]
I've done this using https://github.com/lhotse-speech/lhotse, which has a Whisper model with word-level alignment

**Reply by junwang4** [reply]
I am currently using https://github.com/lowerquality/gentle to do the word-level alignment for the tortoise-tts audio output. It is pretty fast on CPU and the result is good enough. If tortoise can output the word-level alignment as a by-product, that would save me the additional step. BTW, what is your experience of using Lhotse? Thanks!
